### ISSUE TITLE
Bad usability of "file is locked" message in upload

### ISSUE THREAD

**Issue by Spartachetto** [nextcloud/server#20223]
**Nextcloud version:** (see Nextcloud admin page)
18.0.3

**Updated from an older Nextcloud/ownCloud or fresh install:**
Updated

I uploaded a directory with multiple subdirectories.
I received tens of notifications for locked files.

![problemWithLockedfiles](https://user-images.githubusercontent.com/12546982/77854978-d3092080-71ed-11ea-9c44-4b62aaa76093.png)

The notifications were correct, yet there are three problems:

1. the notifications were misaligned on the left, you can notice it by looking at the red vertical stripes. So a little bug in the interface

2.  I had to dismiss all the notifications by hand, and I said there were tens of them. I think that a "_x files locked_" message would be better

3. I did not have any practical mean to save the list of files with problems for a successive check. A way of saving all the notifications in a file/note/whatever... would allow the user to review easily each problem afterwards

**Comment by Spartachetto** [comment]
Maybe @jancborchardt  as this is UI...?

**Comment by jancborchardt** [comment]
Yes, agree with all your points. Also it seems they extend too much in width – they should not overlap the whole width of the interface but instead there should be line-breaks in the text before.

cc @juliushaertl @ma12-co @skjnldsv for the notification component.

**Comment by juliushaertl** [comment]
Yes, would need some additional css to set a max-width https://github.com/nextcloud/nextcloud-dialogs/blob/master/styles/toast.scss

**Comment by szaimen** [comment]
Hi, please update to 24.0.8 or better 25.0.2 and report back if it fixes the issue. Thank you!
